### PR TITLE
Fix sqlite db path in parameters.yml.dist

### PR DESCRIPTION
**Pull request by chalasr**
This should have been done in #1032. ~Adding the path to `.gitignore` was done on symfony-demo, I think we can do it here too.~

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,8 @@
 !/var/cache
 /var/cache/*
 !var/cache/.gitkeep
+!/var/data
+/var/data/*
```
Comment (Tobion) [inline .gitignore]: var/data is already ignored by var/. What's the point of this?

Comment (xabbuh) [inline .gitignore]: I agree, this only makes sense if we also commit a `.gitkeep` file inside `var/data/`.

Comment (chalasr) [inline .gitignore]: indeed, reverted

**Comment by fabpot** [comment]
Merging into 3.2 instead.

**Comment by chalasr** [comment]
Rebased on 3.2.

**Pull request closed without merging**